### PR TITLE
Using plugin handler to add other pulsar related tools

### PR DESCRIPTION
**Pull request by zymap**
---

Master Issue: #179

*Motivation*

Using plugin handler to add more pulsar related tools

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -22,13 +22,26 @@ import (
...
 
 func main() {
 	rootCmd := pkg.NewPulsarctlCmd()
+	handler := plugin.NewDefaultPluginHandler(plugin.ValidPluginFilenamePrefixes)
 
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err) // outputs cobra errors
-		os.Exit(-1)
+		err := plugin.HandlePluginCommand(handler, getArgs())
+		if err != nil {
```
[the remark this answers is not in the document]
Comment (zymap) [reply main.go]: Yes. Will fix it.

**Comment by zymap** [comment]
@sijie  Sure. 

@EronWright Yes, it does. I will add the list plugins command in a separate PR. This PR just make the Pulsarctl can use plugins.

**Comment by zymap** [comment]
@sijie @wolfstudy @EronWright  I add a document to introduce how to use the plugin. PTAL. Thanks.

**Comment by sijie** [comment]
@wolfstudy PTAL

**Comment by zymap** [comment]
ping @wolfstudy

**Pull request merged**